backend/coins/eth/updater.go

```diff
@@ -23,6 +24,7 @@ var pollInterval = 5 * time.Minute
 //go:generate moq -pkg mocks -out mocks/balancefetcher.go . Interface
 type BalanceFetcher interface {
 	Balances(ctx context.Context, addresses []common.Address) (map[common.Address]*big.Int, error)
+	rpcclient.Interface
```

Comment:
Maybe it makes sense to add the block number call only to not blow up this interface? Not sure which is better though.